tests/benchmarks/test_join.py

```diff
@@ -54,7 +54,7 @@ def test_join_big_small(small_client, memory_multiplier, configure_shuffling):
 
 
 @pytest.mark.parametrize("persist", [True, False])
-def test_set_index(small_client, persist, memory_multiplier, configure_shuffling):
```

Comment:
These tests with `"tasks"` take ten minutes to run and I think that we don't really care about this system for dataframes any longer?